tests/lib/Output/Generator/_fixtures/testGeneratorHashElement.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <elements>
  <element attribute="attribute value 1">element value 1</element>
- <element attribute="attribute value 2">element value 2</element>
+ <element2 attribute="attribute value 2">element value 2</element2>
```

Comment:
What's the source of/reason for that change?

Reply:
The reason for this is that `startHashElement` does not support multiple elements of the same name, just like Json Generator. The proper way to handle this would be to use `startHashElement` with conjunction of `startList` like it's done in `ContentFieldValidationException` visitor:

```php
        $generator->startHashElement('errorDetails');
        $generator->startList('fields');
```

We are imo safe here with this change because our visitors are both handling Json and Xml and if we had something like:

```php
        $generator->startHashElement('elements');

        $generator->startValueElement('element', 'element value 1', ['attribute' => 'attribute value 1']);
        $generator->endValueElement('element');

        $generator->startValueElement('element', 'element value 2', ['attribute' => 'attribute value 2']);
        $generator->endValueElement('element');

        $generator->endHashElement('elements');
```
JSON output would be also broken.